Calculator/Calculator/Protocol/CalculateItem.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  CalculateItem.swift
+//  Calculator
+//
+//  Created by Serena on 2023/05/29.
+//
+
+protocol CalculateItem {
+    
+}
```

Comment:
```suggestion
protocol CalculateItem {}
```

빈 구현의 경우 중괄호를 붙여주는 것이 일반적인 것 같습니다.

Reply:
네~! 참고하여 수정했습니다 😆